lib/cvss_suite/helpers/extensions.rb

```diff
@@ -16,19 +16,14 @@ class Float
   # Since CVSS 3 all float values are rounded up, therefore this method is used instead of the mathematically correct method round().
 
   def round_up(decimal_paces = 0)
-    (self * 10.0**decimal_paces).ceil / 10.0**decimal_paces
+    ceil(decimal_paces).to_f
   end
 
   ##
   # The “Round up” function in CVSS v3.0 has been renamed Roundup and is now defined more precisely to minimize the possibility of implementations generating different scores due to small floating-point inaccuracies. This can happen due to differences in floating point arithmetic between different languages and hardware platforms.
 
   def roundup
-    output = (self * 100000).round
-    if (output % 10000) == 0
-      return output / 100000.0
-    else
-      return ((output / 10000).floor + 1) / 10.0
-    end
+    round(2).ceil(1).to_f
```

Comment:
I don't understand why we use two different functions for CVSS 3.0 and CVSS 3.1 when the FIRST specifications are identical.